src/components/manage/Widgets/SearchGeoName.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import InlineForm from 'volto-slate/futurevolto/InlineForm';
```

Comment:
Please add `volto-slate` dependency to README and DEVELOP

Reply:
Sure.